examples/js/controls/TransformControls.js

```diff
@@ -134,6 +136,9 @@
 
 						object.name = name;
 
+						//always render on top of the scene (avoid being hidden by transparent objects)
+						object.renderOrder = Infinity;
```

Comment:
Setting `renderOrder` to `Infinity` does not cause an object to be rendered "on top".

This only works in this case because of the `depthTest = false`, `depthWrite = false` and `transparent = true` material settings.

You can either remove the comment, or change it to

    // avoid being hidden by other transparent objects

Reply:
Thanks ! updated